### PR TITLE
Implemented front-end behavior to take in an ICS file from the user and send their unavailability to our back-end.

### PR DESCRIPTION
**Pull request by Shane-McGookey**
The user's unavailability is parsed out by a script written by Jeremy, and I have removed that script from this PR such that Jeremy can add it himself to make sure due credit is given. Back-end code is also done, but will be on a separate branch related to lambda updates.

### CODE REVIEW

**Review by dantebarbieri — APPROVED** [review]
I'm pretty happy with this ngl, but what does @benbeauchamp7 think?

File: src/Components/Profile/ProfileStatusBar.tsx [hunk 1]
```diff
@@ -1,14 +1,107 @@
...
+
+  const handleChange = (event: any) => {
+    const inputICSFile = event.target.files[0];
+    readInputICSFile(inputICSFile);
+
+    // Reset target.value to allow user
+    // to upload a new file if desired. 
+    if (event.target.value !== null) {
+      event.target.value = null;
+    }
```
Comment (benbeauchamp7) [inline src/Components/Profile/ProfileStatusBar.tsx:103]: What is the if statement here for? event.target.value ends up as null either way

File: src/Components/Profile/ProfileStatusBar.tsx [hunk 2]
```diff
@@ -1,14 +1,115 @@
...
+  }
+
   return (
-    <div className="profile-status-bar"> 
-      <button className="blue button">Upload Schedule</button>
+    <div className="profile-status-bar">
+      <input type="file" ref={fileInputRef} onChange={handleChange} style={{ display: "none" }}/>
```
Comment (benbeauchamp7) [inline src/Components/Profile/ProfileStatusBar.tsx]: Could we restrict the file upload to only accept .ics files?

**Review by benbeauchamp7 — APPROVED** [review]
Looks good, take a look at those two comments and make changes if you'd like. I also noticed that the profile image wasn't loading on this branch.

File: src/Components/Profile/ProfileStatusBar.tsx (hunk 2 above)
Comment (dantebarbieri) [reply src/Components/Profile/ProfileStatusBar.tsx]: This is possible, and I'd like for us to make this change. I'm not sure exactly how or I'd leave a suggestion.

File: src/Components/Profile/ProfileStatusBar.tsx (hunk 1 above)
Comment (Shane-McGookey) [reply src/Components/Profile/ProfileStatusBar.tsx:103]: Just a standard habit, can remove if necessary lol.

**Pull request merged**